superset/dashboards/dao.py

```diff
@@ -47,6 +47,15 @@ def validate_update_slug_uniqueness(dashboard_id: int, slug: Optional[str]) -> b
             return not db.session.query(dashboard_query.exists()).scalar()
         return True
 
+    @staticmethod
+    def update_charts_owners(model: Dashboard, commit: bool = True) -> Dashboard:
+        owners = [o for o in model.owners]
```

Comment:
I think it would be more lint-friendly to have `owner` in the list comprehension